spec/compliance_data_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'spec_helper'
+require 'stub_classes'
+
+describe ComplianceData do
+  describe 'initialize' do
+    it 'returns the right thing' do
+      data = ComplianceData.new StubClasses::StubSettings
+      expect(data).to be_an_instance_of ComplianceData
+    end
```

Comment:
I'm not sure that testing `.new` is super useful...maybe rewrite this test to ensure that the settings are accepted/saved in the instance?